examples/spyre_inference_ppc64le.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python3
+import argparse
+import asyncio
+import atexit
+import gc
+import glob
+import io
+import os
+import pathlib
+import signal
+import subprocess
+import sys
+import threading
+import time
+
+import numpy
+from openai import APIConnectionError, OpenAI
+
+# Bump if you make changes to the log file.
+VERSION = 1
+
+
+# Log the interesting blips to logbuf, and tee to stdout
+def log(string):
+    global logbuf
+    print(string)
+    print(string, file=logbuf)
+
+
+def cleanup_pserver():
+    pserver.terminate()
+    pserver.wait()
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d', '--debug', action='store_true')
+parser.add_argument('-l', '--online', action='store_true')
+parser.add_argument('-r', '--remote', action='store_true')
+parser.add_argument('-m',
+                    '--model',
+                    default="/models/granite-3.0-8b-instruct-r241014a/")
+parser.add_argument('-t', '--maxtoken', type=int, default=64)
+parser.add_argument('-n', '--promptlen', type=int, default=256)
+parser.add_argument(
+    '-i',
+    '--prompts',
+    default=
+    "/home/senuser/aiu-fms-testing-utils/tests/resources/prompts/granite/input"
+)
+parser.add_argument('-p', '--port', type=int, default=8000)
+parser.add_argument('-o', '--output', default="./")
+parser.add_argument('-b', '--batch_size', type=int, default=1)
+parser.add_argument('-k', '--keepoutput', action='store_true')
+parser.add_argument('-c', '--cards', type=int, default=0)
+parser.add_argument('--mintoken', type=int, default=0)
+args = parser.parse_args()
+debug = args.debug
+online = args.online
+fms_input_dir = args.prompts
+tmpdir_output_dir = args.output
+batch_size = args.batch_size
+max_tokens = args.maxtoken
+promptlen = args.promptlen
+logbuf = io.StringIO()
+keepoutput = args.keepoutput
+num_cards = args.cards
+min_tokens = args.mintoken
+
+# For offline inferencing.
+os.environ["VLLM_SPYRE_WARMUP_NEW_TOKENS"] = str(max_tokens)
+os.environ["VLLM_SPYRE_WARMUP_BATCH_SIZES"] = str(batch_size)
+os.environ["VLLM_SPYRE_WARMUP_PROMPT_LENS"] = str(promptlen)
+
+# Verify setup is consistent with number of aius
+if not args.remote:
+    if "VLLM_AIU_PCIE_IDS" not in os.environ or os.environ[
+            "VLLM_AIU_PCIE_IDS"] == "":
+        print("source vllm_start.sh before running scripti."
```

Comment:
vllm_start.sh doesn't exist externally here :/

Can we update this to instead mention `/etc/profile.d/ibm-aiu-setup.sh`, which is available in the public base image?